sandbox/grommet-app/src/components/CollapsibleMenu/ExpandableMenuItem.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext } from 'react';
+import { Box, Button, Collapsible, Text, ThemeContext } from 'grommet';
+import { Down, LinkNext, Up } from 'grommet-icons';
+
+export const ExpandableMenuItem = ({
+  item,
+  value,
+  open,
+  expanded,
+  setExpanded,
+  selected,
+  onSelect
+}) => {
+  const { global } = useContext(ThemeContext) as { global: { edgeSize: { medium: string } } };
+  return (
+    <Box key={item} gap="xsmall">
+      <Button
+        label={
+          <Box direction="row" align="center" justify="between">
+            <Text>{item}</Text>
+            {open ? <Up /> : <Down />}
+          </Box>
+        }
+        justify="start"
+        onClick={() => {
+          if (open) {
+            setExpanded(expanded.filter(i => i !== item));
+          } else {
+            setExpanded([...expanded, item]);
+          }
+        }}
+      />
+      <Collapsible open={open} >
+        {value.map((subItem) => {
+          const active = selected.key === subItem.key;
+          return (
+            <Button
+              key={subItem.key}
+              label={
+                <Box direction="row" align="center" justify="between">
+                  <Box direction="row" align="center" gap="xsmall">
+                    <Box width={global.edgeSize.medium}>
```

Comment:
I'm comfortable with this. Would is make sense to have the width of this be set to `medium.hpe.size.icon.medium`? Meaning we're reserving space directly for the icon's dimension.

Reply:
done.